docs/api/get-started.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: page
+---
+
+# Getting Started with the To-Do-Service API
+
```

Comment:
Try not to have headings that are followed immediately by sub headings.
If there's no intervening text, can you collapse them in to a single heading?

If this is a "getting started" topic, then they've already seen the value proposition and are ready to get started.

You could, for example, focus on the "Getting started" theme and dive right into the quick start with only a brief orientation sentence such as line 11 below.